example/youtube/README.md

```diff
@@ -1,15 +1,24 @@
-# `youtube_example`
+# `Youtube Example`
 
 List top 5 youtube videos.
 
 ```shell
-$ cargo run
+CURRENT_DIR=`pwd`
```

Comment:
ah, so you assume that any user would clone this repository, generate code as was done before, and copy it to their crate?

That's fair, but I don't quite see why we need to put the python generator into the crate that is only supposed to contain support code. Is it intentional, or would you leave out the python generator prior to publishing?

Reply:
yes, like we use it now but much simpler.
we can separate the generator directory.
it can be like:
`crate`
`generator`
top level directories in repo.

Reply:
Yes, like we used it before but much simpler.

It wasn't intentional but having generator in the cloned crate, users can keep the generated codes up to date and also put it on their build process, or they can delete it...